Makefile

```diff
@@ -194,6 +194,14 @@ build/undefined_syms.txt: undefined_syms.txt
 clean:
 	$(RM) -r $(ROM) $(ELF) build
 
+distclean: clean
+	$(RM) -r $(TEXTURE_BIN_DIRS)
+	$(RM) -r baserom/
+	$(MAKE) -C tools distclean
+
+nuke: distclean
+	@echo "Nuked."
```

Comment:
I agree with Fig that this alias seems unnecessary

Reply:
Removed.